docs/program/apis/session-manager.md

```diff
@@ -126,6 +129,113 @@ disableSessions: true
 {{% /tab %}}
 {{% /tabs %}}
 
-This option allows you to have full control over sessions management.
+This option allows you to have full control over session management.
 After disabling the client, you must now manage each of your sessions manually with the session management API.
 You can do this with Viam's [client SDKs](https://pkg.go.dev/go.viam.com/rdk/session).
+
+## API
+
+The `SessionManager` API built-in to the [Robot API](/program/apis/robot/) supports the following methods:
```

Comment:
I think this needs documentation on `NewSessionManager()` in robot API to be functional-- sme review may help clear up some confusion on my end

Reply:
Added, then removed,`NewSessionManager()` instantiation of `mySessionManager`-- unsure whether that should be in there or not. How would a user most simply request to the default session manager API with what we have set up in the RDK? @dgottlieb